include/NovelRT/NovelRunner.h

```diff
@@ -42,7 +42,7 @@ namespace NovelRT {
      * @param windowTitle The title of the window created for NovelRunner.
      * @param targetFrameRate The framerate that should be targeted and capped.
      */
-    explicit NovelRunner(int displayNumber, const std::string& windowTitle = "NovelRTTest", uint32_t targetFrameRate = 0, bool transparency = false);
+    explicit NovelRunner(int displayNumber, const std::string& windowTitle = "NovelRTTest", uint32_t targetFrameRate = 0, bool transparency = false, NovelRT::Windowing::WindowMode = NovelRT::Windowing::WindowMode::Windowed);
```

Comment:
There's a merge conflict here - `int displaynumber` should be `int32_t displaynumber`